src/pybind/mgr/dashboard/frontend/src/app/shared/api/cephfs.service.spec.ts

```diff
@@ -1,20 +1,51 @@
-import { HttpClientModule } from '@angular/common/http';
-import { inject, TestBed } from '@angular/core/testing';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { TestBed } from '@angular/core/testing';
 
 import { CephfsService } from './cephfs.service';
 
 describe('CephfsService', () => {
+  let service: CephfsService;
+  let httpTesting: HttpTestingController;
+
   beforeEach(() => {
     TestBed.configureTestingModule({
-      imports: [HttpClientModule],
+      imports: [HttpClientTestingModule],
       providers: [CephfsService]
     });
+
+    service = TestBed.get(CephfsService);
+    httpTesting = TestBed.get(HttpTestingController);
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
+
+  it('should call list', () => {
+    service.list().subscribe();
+    const req = httpTesting.expectOne('api/cephfs');
+    expect(req.request.method).toBe('GET');
+    httpTesting.verify();
```

Comment:
Could you please put `httpTesting.verify()` into `afterEach`?
(This can be done an many other files as well)

Reply:
I would prefer to keep the verification inside each test.
Otherwise you won't know exactly which test failed and have to check all of them.
That is not a problem if you only have 1 test in a file, but if you have 10+ it can get tricky.

Reply:
It's described [here](https://angular.io/guide/http#expecting-and-answering-requests) as a common way to do it. Please amend it.

Reply:
I think I would also go with moving the `httpTesting.verify()` into an `afterEach()` function.